{"metadata":{"role":"symbol","fragments":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorMessage"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"}],"roleHeading":"Instance Property","symbolKind":"property","externalID":"s:10StytchCore0A8APIErrorC12errorMessageSSvp","title":"errorMessage","modules":[{"name":"StytchCore"}]},"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"platforms":["iOS"],"tokens":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"errorMessage","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"}]}]}],"variants":[{"paths":["\/documentation\/stytchcore\/stytchapierror\/errormessage"],"traits":[{"interfaceLanguage":"swift"}]}],"sections":[],"abstract":[{"text":"The message associated with the error.","type":"text"}],"identifier":{"url":"doc:\/\/stytch-ios.StytchCore\/documentation\/StytchCore\/StytchAPIError\/errorMessage","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/stytch-ios.StytchCore\/documentation\/StytchCore","doc:\/\/stytch-ios.StytchCore\/documentation\/StytchCore\/StytchAPIError"]]},"schemaVersion":{"minor":3,"patch":0,"major":0},"kind":"symbol","references":{"doc://stytch-ios.StytchCore/documentation/StytchCore":{"type":"topic","kind":"symbol","url":"\/documentation\/stytchcore","identifier":"doc:\/\/stytch-ios.StytchCore\/documentation\/StytchCore","abstract":[{"text":"Provides native iOS access to the Stytch SDK methods for ultimate flexibility.","type":"text"}],"title":"StytchCore","role":"collection"},"doc://stytch-ios.StytchCore/documentation/StytchCore/StytchAPIError":{"type":"topic","role":"symbol","identifier":"doc:\/\/stytch-ios.StytchCore\/documentation\/StytchCore\/StytchAPIError","navigatorTitle":[{"text":"StytchAPIError","kind":"identifier"}],"abstract":[{"text":"Error class representing an error within the Stytch API.","type":"text"}],"url":"\/documentation\/stytchcore\/stytchapierror","fragments":[{"text":"class","kind":"keyword"},{"text":" ","kind":"text"},{"text":"StytchAPIError","kind":"identifier"}],"kind":"symbol","title":"StytchAPIError"},"doc://stytch-ios.StytchCore/documentation/StytchCore/StytchAPIError/errorMessage":{"type":"topic","fragments":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"errorMessage","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"}],"abstract":[{"type":"text","text":"The message associated with the error."}],"title":"errorMessage","identifier":"doc:\/\/stytch-ios.StytchCore\/documentation\/StytchCore\/StytchAPIError\/errorMessage","url":"\/documentation\/stytchcore\/stytchapierror\/errormessage","kind":"symbol","role":"symbol"}}}